### PR TITLE
Feature/sm 2

### PR DESCRIPTION
**Pull request by rotarcatalin-stack**
Feature/SM-2 Implemented the View Students Story

### CODE REVIEW

**Comment by rotaraugustin** [comment]
Feature/SM-2 Implemented the View Students Story

File: src/loose/oose/fis/lab/student/manager/Main.java [hunk 1]
```diff
@@ -9,14 +9,13 @@
 public class Main extends Application {
 
     @Override
-    public void start(Stage primaryStage) throws Exception{
-        Parent root = FXMLLoader.load(getClass().getResource("sample.fxml"));
-        primaryStage.setTitle("Hello World");
-        primaryStage.setScene(new Scene(root, 300, 275));
+    public void start(Stage primaryStage) throws Exception {
+        Parent root = FXMLLoader.load(getClass().getResource("fxml/view-students.fxml"));
+        primaryStage.setTitle("Student Manager");
```
Comment (rotarcatalin-stack) [inline src/loose/oose/fis/lab/student/manager/Main.java:14]: suggestion
change title to FIS - Student Manager

**Review by rotaraugustin — APPROVED** [review]
cool approved

File: src/loose/oose/fis/lab/student/manager/Main.java [hunk 2]
```diff
@@ -9,14 +9,13 @@
...
+        primaryStage.setScene(new Scene(root, 600, 400));
```
Comment (rotaraugustin) [inline src/loose/oose/fis/lab/student/manager/Main.java:15]: change

**Pull request merged**